webapp-django/crashstats/crashstats/decorators.py

```diff
@@ -79,8 +79,8 @@ def pass_default_context(view):
     """
     @functools.wraps(view)
     def inner(request, *args, **kwargs):
-        product = kwargs.get('product', None)
-        versions = kwargs.get('versions', None)
+        product = kwargs.get('product', request.GET.get('product', None))
+        versions = kwargs.get('versions', request.GET.get('versions', None))
```

Comment:
This was half of the issue. The problem here was that this code used to pull the product and version from the url *path*, but the topcrashers report (and exploitability report) have that information in the url *querystring*. So I fixed this to try one, then the other.